torch/_cuda_sanitizer/enable.py

```diff
@@ -0,0 +1,3 @@
+from torch._cuda_sanitizer._sanitizer import CUDASanitizer
+
+cuda_sanitizer = CUDASanitizer()
```

Comment:
It's better to not use a side-effectful module; otherwise tools that remove dead Python imports will remove this tool.